src/pattern.class.js

```diff
@@ -95,6 +95,7 @@
         repeat: this.repeat,
         offsetX: toFixed(this.offsetX, NUM_FRACTION_DIGITS),
         offsetY: toFixed(this.offsetY, NUM_FRACTION_DIGITS),
+        patternTransform:  this.patternTransform
```

Comment:
patternTransform: this. patternTransform ? this. patternTransform.concat() : null,

this allow to avoid sending out a reference to the real matrix that could be changed by other function later.
A sort of shallow protection on mutations.

You do not expect your object to to influence rendering once exported.

Reply:
great version ( at this version you broke video scalling ;))

Reply:
?